datafusion/spark/src/function/string/mod.rs

```diff
@@ -46,6 +48,11 @@ pub mod expr_fn {
         "Returns the ASCII character having the binary equivalent to col. If col is larger than 256 the result is equivalent to char(col % 256).",
         arg1
     ));
+    export_functions!((
+        elt,
+        "Returns the n-th input, e.g., returns input2 when n is 2. The function returns NULL if the index exceeds the length of the array and spark.sql.ansi.enabled is set to false.",
```

Comment:
Do we have a way to set `spark.sql.ansi.enabled`? If not then I don't think we include that detail in the description